src/main.rs

```diff
@@ -327,6 +328,11 @@ fn main() -> std::io::Result<()> {
         return Ok(());
     }
 
+    if options.contains(&"--version".to_string()) || options.contains(&"-v".to_string()) {
+        print_version();
```

Comment:
Might make more sense to do that `option_env` call here instead of storing the version in a variable